chapters/implementation/sections/game_objects_management/subsections/scene_class.tex

```diff
@@ -1,18 +1,16 @@
-\section{Scene}
-
+\subsection{\texttt{Scene} class}
 The \texttt{Scene} class is responsible for storing all the objects in the game the player can interact with.
 It is also responsible for disposing of them.
 Here is the full list of objects stored in a \texttt{Scene} instance:
 
 \begin{itemize}
```

Comment:
I don;t like they way enumeration looks both here and in controllers. 10 words take a whole page